static/js/reducers/posts.js

```diff
@@ -0,0 +1,15 @@
+// @flow=
```

Comment:
Yeah, but I'm not sure how to configure that. Should I just nix these here then? I'm currently using these as a hack for dummy data on my other branch.